apps/aecore/src/aec_hard_forks.erl

```diff
@@ -141,6 +141,8 @@ protocols_from_network_id(<<"local_iris_testnet">>) ->
      %%, ?LIMA_PROTOCOL_VSN     => Excluded for testing new protocol
      , ?IRIS_PROTOCOL_VSN     => 1
      };
+protocols_from_network_id(<<"ae_dev">>) ->
```

Comment:
What's the purpose of this hardcoded network id ?

Reply:
it would be nice to be able to provide a custom one at some point.

Reply:
I believe you already can. But you have to specify the consensus properties in detail then. Also, the on-demand consensus mode is really not meant for networking.